DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -2869,11 +2869,13 @@ internal async Task<DiscordMessage> EditWebhookMessageAsync(ulong webhook_id, st
         {
             builder.Validate(true);
 
+            var mentions = builder.Mentions != null ? new DiscordMentions(builder.Mentions, builder.Mentions?.Any() ?? false) : null;
```

Comment:
`builder.Mentions?.Any() ?? false` should just be `builder.Mentions.Any()` since null is already checked, though it shouldn't have a functional impact.
Otherwise, LGTM.